src/main/java/io/hops/util/Hops.java

```diff
@@ -1053,4 +1054,23 @@ public static String getElasticAuthorizationToken()
     configs.put("es.resource", getElasticIndex(index));
     return configs;
   }
+  
+  /**
+   * Set environment variables
+   * @param key
+   * @param value
+   */
+  // TODO: Ahmad: A hack to set env variable for debugging! Must be set somewhere in glassfish?
+  public static void setEnv(String key, String value) {
```

Comment:
I thought in the end we would set the environment variables, except spark, and we would point users to documentation how to fetch the AWS tokens.

Reply:
This is setting the environment variables.

Reply:
Sorry my comment was wrong, I meant to say that I thought we would not set the environment variables for java/scala, but let users do it for their applications as needed by pointing them to documentation. I'm not sure how safe is to set the environment variables like this based on the discussion here https://stackoverflow.com/questions/318239/how-do-i-set-environment-variables-from-java

It will update only the in-memory copy of JVM's environment variables, but this low-level code could mess with other environment variables of the JVM as well.